.github/workflows/tests-ces-migrate.yaml

```diff
@@ -0,0 +1,171 @@
+        with:
+          ref: ${{ github.base_ref || github.ref }}
+          persist-credentials: false
+
+      - name: Set Environment Variables
+        uses: ./.github/actions/set-env-variables
+
+      - name: Checkout
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+        with:
+          persist-credentials: false
+
+      - name: Set image tag
+        id: sha
+        run: |
+          if [ "${{ github.event.pull_request.head.sha }}" != "" ]; then
+            echo sha=${{ github.event.pull_request.head.sha }} >> $GITHUB_OUTPUT
+          else
+            echo sha=${{ github.sha }} >> $GITHUB_OUTPUT
+          fi
+
+      - name: Create kind cluster
+        uses: helm/kind-action@0025e74a8c7512023d06dc019c617aa3cf561fde # v1.10.0
+        with:
+          version: ${{ env.KIND_VERSION }}
+          node_image: ${{ env.KIND_K8S_IMAGE }}
+          kubectl_version: ${{ env.KIND_K8S_VERSION }}
+          config: ${{ env.KIND_CONFIG }}
+          wait: 0 # The control-plane never becomes ready, since no CNI is present
+
+      - name: Wait for images to be available
+        timeout-minutes: 30
+        shell: bash
+        run: |
+          for image in cilium-ci operator-generic-ci hubble-relay-ci ; do
+            until docker manifest inspect quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/$image:${{ steps.sha.outputs.sha }} &> /dev/null; do sleep 45s; done
+          done
+
+      - name: Set up install variables
+        id: cilium-config
+        uses: ./.github/actions/cilium-config
+        with:
+          image-tag: ${{ steps.sha.outputs.sha }}
+          chart-dir: 'install/kubernetes/cilium'
+          ipv6: false
+          egress-gateway: false # Currently incompatible with CES
+          mutual-auth: false
+          misc: 'bpfClockProbe=false,cni.uninstall=false'
+
+      - name: Install Cilium CLI
+        uses: cilium/cilium-cli@bfa4f93a41da410a1b4c9373c1694ca6d5c35ade # v0.16.6
+        with:
+          repository: ${{ env.CILIUM_CLI_RELEASE_REPO }}
+          release-version: ${{ env.CILIUM_CLI_VERSION }}
+          ci-version: ${{ env.cilium_cli_ci_version }}
+          binary-name: cilium-cli
+          binary-dir: ./
+
+      - name: Install Cilium
+        id: install-cilium
+        run: |
+          ./cilium-cli install ${{ steps.cilium-config.outputs.config }}
+
+      - name: Wait for Cilium status to be ready
+        run: |
+          ./cilium-cli status --wait
+          kubectl get pods --all-namespaces -o wide
+          mkdir -p cilium-junits
+          kubectl -n kube-system exec daemonset/cilium -c cilium-agent -- cilium-dbg status
+
+      - name: Enable CiliumEndpointSlice & Test
+        uses: ./.github/actions/conn-disrupt-test
+        with:
+          job-name: ces-enable
+          operation-cmd: |
+            kubectl patch -n kube-system configmap cilium-config --type merge --patch '{"data":{"enable-cilium-endpoint-slice":"true"}}'
+
```

Comment:
I believe the usage of `conn-disrupt-test` needs adjustment for the changes in https://github.com/cilium/cilium/pull/32930.

Reply:
Yep, just mentioned the same: https://cilium.slack.com/archives/C069SNDMLA2/p1718286768251579

Reply:
The path filter should also be modified so that these changed are detected immediately.